csfieldguide/templates/interactives/lzss-compression.html

```diff
@@ -0,0 +1,48 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container">
+    <div class="row">
+      <div class="col s12 l6 offset-l3">
```

Comment:
This isn't the right syntax anymore. I think instead you would use col-sm-12 col-lg-6 offset-lg-3. See https://getbootstrap.com/docs/4.1/layout/grid/#auto-layout-columns

Reply:
Will have to go through other interactives and check for this too